scalyr_agent/tests/url_monitor_test.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2014 Scalyr Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------
+#
+# author: Saurabh Jain <saurabh@scalyr.com>
+
+__author__ = 'saurabh@scalyr.com'
+
+import unittest
+import mock
+from scalyr_agent.builtin_monitors.url_monitor import UrlMonitor
+from scalyr_agent.json_lib.objects import JsonArray, JsonObject
+
+
+class UrlMonitorTestRequest(unittest.TestCase):
+    """
+    Tests the formation of the request for monitoring a URL
+    """
+
+    def setUp(self):
+        self.legit_headers = JsonArray()
+        self.legit_headers.add(JsonObject({'header': 'header_foo', 'value': 'foo'}))
+        self.legit_headers.add(JsonObject({'header': 'header_bar', 'value': 'bar'}))
+
+    def tearDown(self):
+        pass
+
+    def testGetRequestNoHeaders(self):
```

Comment:
style question.  We are using snake case for all of our method names (for the most part).  Do you feel that tests should be treated different because the of the influence of junit (`setUp`, etc).

Reply:
my preference is to keep tests n the same case as the rest of the code (snake_case)- yeah the `setUp`, `tearDown` throws it off but we can live with it?